frontend/src/bundles/users/components/user-card/helpers/get-initials.helper.ts

```diff
@@ -0,0 +1,12 @@
+const getInitials = (fullName: string | undefined): string => {
+    if (fullName) {
+        const [firstName, lastName] = fullName.trim().split(/\s+/);
+        const firstInitial = firstName?.charAt(0).toUpperCase();
+        const secondInitial = lastName?.charAt(0).toUpperCase() ?? '';
+
+        return `${firstInitial}${secondInitial}`;
+    }
+    return 'FN';
+};
+
+export { getInitials };
```

Comment:
Consider using early return for better readability
```suggestion
const getInitials = (fullName: string | undefined): string => {
    if (!fullName) {
        return 'FN';
    }
    
    const [firstName, lastName] = fullName.trim().split(/\s+/);
    const firstInitial = firstName?.charAt(0).toUpperCase();
    const secondInitial = lastName?.charAt(0).toUpperCase() ?? '';

    return `${firstInitial}${secondInitial}`;
};

export { getInitials };
```